test/turn-test.js

```diff
@@ -0,0 +1,61 @@
+import chai from 'chai';
+import Turn from '../src/Turn'
+import Player from '../src/Player';
+import Round from '../src/Round';
+import Game from '../src/Game';
+import Puzzle from '../src/Puzzle';
+
+const expect = chai.expect;
+
+describe('Turn', function () {
+    let player;
+    let round;
+    let turn;
+    let game;
+    let puzzle;
+
+    beforeEach(function () {
+        player = new Player('Julian')
+        game = new Game('Julian', 'Lacy', 'Brandy')
+        round = new Round(game)
+        turn = new Turn(round, 'Julian');
+        puzzle = new Puzzle()
+        puzzleAnswer = puzzle.correctAnswer.split()
+        // console.log(puzzleAnswer)
+        it('should be a function', () => {
```

Comment:
You shouldn't have any `it` blocks within the `beforeEach`. These tests will run for every test.